python/sglang/srt/layers/moe/topk.py

```diff
@@ -250,10 +251,12 @@ def select_experts(
     custom_routing_function: Optional[Callable] = None,
     correction_bias: Optional[torch.Tensor] = None,
     torch_native: bool = False,
+    routed_scaling_factor: float = 2.5,
 ):
     n_share_experts_fusion = 0
     if global_server_args_dict["n_share_experts_fusion"] is not None:
         n_share_experts_fusion = global_server_args_dict["n_share_experts_fusion"]
+        routed_scaling_factor = global_server_args_dict["routed_scaling_factor"]
```

Comment:
hmm, wondering whether it is good to have it as a global variable... what about passing in as a normal variable, especially because select_experts does add this arg